src/rpc/blockchain.cpp

```diff
@@ -1182,7 +1182,8 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
     obj.push_back(Pair("pruned",                fPruneMode));
 
     const Consensus::Params& consensusParams = Params().GetConsensus();
-    CBlockIndex* tip = chainActive.Tip();
+    const auto tip = chainActive.Tip();
+    assert(tip);
```

Comment:
Is it ok to assert here, given that this is a RPC call? I was thinking that maybe explicit handling of the null case would be better, to avoid asserting. What do you think?

Reply:
@sebastianrusu Thanks for having a look. Actually, I had a closer look now and most of the code assumes `chainActive.Tip()` is not nullptr (even in the same function). If you are OK, maybe let's just close that PR without merging it? Sorry for the noise.